src/plugins/data_source_management/public/components/breadcrumbs.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
```

Comment:
since this is a new file, let's use the consistent header. -- u can simply delete the header and run the node fix script.